app.js

```diff
@@ -1,4 +1,64 @@
-import {app} from "mu";
-app.get('/', function( req, res ) {
-  res.send('hello world')
+import { app, update, query, sparqlEscapeUri, errorHandler } from 'mu';
+
+app.get('/', function (req, res) {
+  res.send('Hello mu-javascript-template');
 });
+
+app.delete('/:id', async function (req, res) {
+
+  const meetingId = "http://data.lblod.info/id/zittingen/" + req.params.id;
```

Comment:
I think it's safer (and more compliant with semanticworks) to look up the uri based on the mu:uuid. We might change the baseURI at some point...